Bottom: 361e65284b10c5561ef26ffba99af8944e6e1c06
Top:    e0aa685b07734b39aa9cbb4a104c96fe28d370f8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:58 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index d9280fa..907c3e1 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ void ghid_set_lock_effects (hidGC gc, AnyObjectType *object);
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
